doc/api/util.md

```diff
@@ -257,6 +257,25 @@ intended as a debugging tool. Some input values can have a significant
 performance overhead that can block the event loop. Use this function
 with care and never in a hot code path.
 
+## util.getSystemErrorName(err)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `err` {number}
+* Returns: {string}
+
+Returns the string name for a numeric error code that comes from a Node.js API.
+The mapping between error codes and error names is platform-dependent.
+See [Common System Errors][] for the names of common errors.
+
+```js
+fs.access('file/that/does/not/exist', (err) => {
+  const name = util.getSystemErrorName(err.errno);
+  console.error(name);  // ENOENT
```

Comment:
Suggestion: It might be good to also show the `err.errno` here to outline the difference.

Reply:
@Bridge The numeric code is platform-dependent so I am not sure if it's OK to just show the value of a particular platform or should I need to show all the possible values...

Reply:
I guess you meant me ;-)

And true, I forgot about that. Hm, either a "e.g. on platform xyz" or just land it as is. What ever you think is best.